src/components/grid/grid.scss

```diff
@@ -2,21 +2,31 @@
 
 /**
  * @prop --lime-grid-area: Grid layout
- * @prop --lime-grid-columns: Number of columns in the grid, defaults to 8
+ * @prop --lime-grid-columns: Number of columns in the grid, defaults to 4
  * @prop --lime-grid-cell-height: Height of grid cells, defaults to 150 pixels
- * @prop --lime-grid-cell-width: Minimum width of grid cells, defaults to 150 pixels
+ * @prop --lime-grid-gutter: width of the gutter between columns and rows, defaults to 16px - see https://material.io/design/layout/responsive-layout-grid.html#breakpoints for guidelines
  */
 
+:host {
+    display: block;
+    position: relative;
+}
+
+:host([hidden]) {
+    display: none;
+}
+
 slot {
     display: grid;
     grid-template-areas: var(--lime-grid-area, '');
     grid-template-columns: repeat(
-        var(--lime-grid-columns, 8),
-        minmax(var(--lime-grid-cell-width, pxToRem(150)), 1fr)
+        var(--lime-grid-columns, 4),
+        minmax(0, 1fr)
     );
-    grid-gap: pxToRem(15);
-    grid-auto-flow: dense;
+    grid-gap: var(--lime-grid-gutter, pxToRem(16));
+    grid-auto-flow: row dense;
     grid-auto-rows: var(--lime-grid-cell-height, pxToRem(150));
```

Comment:
Considering the columns are now rather narrow, we might want the minimum size of the rows to be less than 150px.

Also, we need to make sure this actually sets the row height properly. I think it probably just sets the minimum height. If we need to change it, we should definitely change it before merging this, since that would be another breaking change.